.github/actions/waffles/waffles.py

```diff
@@ -123,6 +123,8 @@ def _get_flask_endpoints(flask_app: Flask) -> List[URL]:
         endpoint = URL(extra_endpoint)
         endpoints.append(endpoint)
 
+    endpoints.remove("/<path:path>")
```

Comment:
We might want to check if the item exists before removing, or a safe way to remove without raising a `ValueError` [per documentation](https://docs.python.org/3/tutorial/datastructures.html).